plugins/essif-lab/src/Integrations/WordPress.php

```diff
@@ -59,10 +61,32 @@ private function registerModelSaveHandler(Model $model): void {
 		$this->utility->call(WP::ADD_ACTION, $hook, function ($_, $post) use ($hook) {
 			$namespace = $this->application->getNamespace();
 			if (array_key_exists($namespace, $_POST)) {
-				$new = $this->prepareModelSaveData($_POST[$namespace], $post);
-				$this->utility->call(WP::REMOVE_ALL_ACTIONS_AND_EXEC, $hook, function () use ($post, $new) {
-					$this->executeModelSave($post, $new);
-				});
+				$namespace_data = $_POST[$namespace];
+				if (array_key_exists(Constants::FIELD_TYPE_SIGNATURE, $namespace_data)) {
+					$newData = $this->prepareModelSaveData($namespace_data, $post);
+					$this->utility->call(WP::REMOVE_ALL_ACTIONS_AND_EXEC, $hook, function () use ($post, $newData) {
+						$this->executeModelSave($post, $newData);
+					});
+				} elseif (array_key_exists(Constants::ACTION_NAME_ADD_RELATION, $namespace_data) && !array_key_exists(Constants::ACTION_NAME_REMOVE_RELATION, $namespace_data)) {
+					$relation_post_type = $namespace_data[Constants::ACTION_NAME_RELATION_ACTION];
```

Comment:
Bij het aanmaken van een nieuwe post zonder het klikken van de add of remove button krijg ik:

Notice: Undefined index: relation_action in ../hbo-ict-inno-s2-tno/plugins/essif-lab/src/Integrations/WordPress.php on line 71